Formula/ion-cli.rb

```diff
@@ -26,25 +26,11 @@ def install
   end
 
   test do
-    if build.head?
-      # Make sure that this version is pointing to HEAD
-      assert_match("HEAD", "#{self.version}")
-      # Head should support all features of `ion-cli`
-      # Verify if `generate` subcommand exist on `beta` (`generate` is an experimental feature under `ion-cli`)
-      assert_match("generate", shell_output("ion beta generate --help"))
-      # Make an Ion schema file with simple type definition
-      (testpath/"example.isl").write "type::{ name: foo, type: int }"
-      # Generate code based on above schema file and assert that the exit status is 0 (successful)
-      shell_output("ion beta generate --directory #{testpath} --schema example.isl --language java --namespace org.example")
-    else
-      # Make sure that `ion --version` outputs the expected version number
-      assert_match("ion #{self.version}", shell_output("ion --version"))
```

Comment:
Did you mean to remove this step?

Reply:
Yes. I don't really see the utility in that particular test. We just need something to ensure that it built correctly and can run. The `ion cat` test should work regardless of whether we're building the latest released version or we're building HEAD, but testing `ion --version` could return a different output if we're building using HEAD.

I'm open to re-adding it if you think it is useful.

Reply:
> The ion cat test should work regardless of whether we're building the latest released version or we're building HEAD, but testing ion --version could return a different output if we're building using HEAD.
Ah right. Thats true. I think we should still keep the `generate` test for now until we remove the feature flag because it ensures that the feature flag commands are working with HEAD.

Reply:
> I think we should also keep the generate test for now until we remove the feature flag

https://github.com/amazon-ion/ion-cli/pull/132